src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -15,7 +16,7 @@ namespace System.Globalization
     public partial class CompareInfo
     {
         [NonSerialized]
-        private Interop.Globalization.SafeSortHandle _sortHandle;
+        private Interop.Globalization.SafeSortHandle? _sortHandle;
```

Comment:
this is still non-nullable but wasn't able to use workaround for this because on Windows same field is struct and the constructor is in the OS independent code

Reply:
can we initialize it to `!null` here instead?